<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>171</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="softground_label">
        <property name="text">
         <string>Soft Ground Thickness</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="softground_lineedit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="clay_label">
        <property name="text">
         <string>Clay Layer Thickness</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="clay_lineedit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="silt_label">
        <property name="text">
         <string>Silt Layer Thickness</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="silt_lineedit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sand_label">
        <property name="text">
         <string>Sand Layer Thickness</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="sand_lineedit"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="apply_pushbutton">
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="cancel_pushbutton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
